src/platform/qt/ts/mgba-fr.ts

```diff
@@ -2514,115 +2514,115 @@ Game Boy Advance est une marque de fabrique enregistré par Nintendo Co., Ltd.</
         <location filename="../IOViewer.cpp" line="866"/>
         <location filename="../IOViewer.cpp" line="879"/>
         <source>R</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">R</translation>
     </message>
     <message>
         <location filename="../IOViewer.cpp" line="867"/>
         <location filename="../IOViewer.cpp" line="880"/>
         <source>L</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">L</translation>
     </message>
     <message>
         <location filename="../IOViewer.cpp" line="882"/>
         <source>Condition</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Condition</translation>
     </message>
     <message>
         <location filename="../IOViewer.cpp" line="886"/>
         <source>SC</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">SC</translation>
     </message>
     <message>
         <location filename="../IOViewer.cpp" line="887"/>
         <source>SD</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">SD</translation>
     </message>
     <message>
         <location filename="../IOViewer.cpp" line="888"/>
         <source>SI</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">SI</translation>
     </message>
     <message>
         <location filename="../IOViewer.cpp" line="889"/>
         <source>SO</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished"><SO/translation>
```

Comment:
Please fix this winding up in the wrong part of the line.

Reply:
I just realized I can do that myself, so I've done it and we'll see if the build passes now.